### DISCUSSION TITLE
Do you use a chat platform?

### DISCUSSION THREAD

**Discussion by n0nag0n** [mako-framework/framework#306] [General]
Do you guys have something like Discord, Gitter, Slack, etc? Didn't see anything obvious anywhere. If you don't have one, I'd recommend https://element.io/ as it's free, they don't make you pay for your message history and has some good features. We just recently migrated over to it from slack and enjoy it so far! You also can have your pick of client as it's based on the Matrix communication network.

**Comment by 1463836** [comment]
邮件已收到，我将尽快给您回复。
